### ISSUE TITLE
if sensor not registered but data sent anyway server crashes

### ISSUE THREAD

**Issue by Nixes** [Nixes/iotHub#26]
```
value was: 19.1
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'data' of null
    at /home/callum/iotHub/routes/api.js:204:15
    at Query.<anonymous> (/home/callum/iotHub/node_modules/mongoose/lib/model.js:3324:16)
    at /home/callum/iotHub/node_modules/kareem/index.js:259:21
    at /home/callum/iotHub/node_modules/kareem/index.js:127:16
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

```

**Comment by Nixes** [comment]
hopefully fixed by b3916338de05e8d2b6094e885f291e6e40757055

**Comment by Nixes** [comment]
actually fixed by 17b3c0617563a529e453c42690354c89aeb99695
